### DISCUSSION TITLE
GitHub Education incorrectly identifies my school from my school mail.

### DISCUSSION THREAD

**Discussion by anhquoc2010** [community/community#64580] [GitHub Education]
GitHub Education incorrectly identifies my school from my school mail. My school is not the school in image. What should I do to reverify GitHub Education?

![Screenshot_2023-08-22-14-06-43-453-edit_com.microsoft.emmx.jpg](https://github.com/community/community/assets/87579077/ca10a41a-8632-4fef-a36f-45dec1e52a20)

**Comment by yilmazemrecom** [comment]
Please add your student email address to your GitHub account and try again

**Reply by anhquoc2010** [reply]
I added it to the account and it appeared as shown in the picture, automatically determining my school incorrectly.

**Reply by yilmazemrecom** [reply]
write to [github support](https://docs.github.com/en/support/contacting-github-support)

**Comment by huynhtrongkhoa** [comment]
I have the same problem with u, I can't register for a student pack because the automatically selected school is wrong.
![image](https://github.com/community/community/assets/109841268/39e19314-8bbb-4ff2-9140-d162da7dd2c4)
